<launch>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen"
        clear_params="true">
        <rosparam file="$(find my_robot_navigation)/param/costmap_common_params.yaml" command="load"
            ns="global_costmap" />
        <rosparam file="$(find my_robot_navigation)/param/costmap_common_params.yaml" command="load"
            ns="local_costmap" />
        <rosparam file="$(find my_robot_navigation)/param/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find my_robot_navigation)/param/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find my_robot_navigation)/param/base_local_planner_params.yaml"
            command="load" />
    </node>
    <!-- 启动控制器管理器 -->
    <node name="car_controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" ns="/car"
        args="joint_state_controller diff_drive_controller arm_controller" />
</launch>